src/test/java/PanelHeaderLinkTest.java

```diff
@@ -20,7 +20,8 @@ public void setUp() {
     }
 
     @BeforeMethod
-    public void beforeMethod() {driver.get(HOMEPAGE_URL);
+    public void beforeMethod() {
+        driver.get(HOMEPAGE_URL);
```

Comment:
In this class in 2 tests methods expected urls should be replace with login page url and create page url defined in PageUrl class.